tests/test_mirror.py

```diff
@@ -235,7 +235,7 @@ def remove_fdb(self, vlan, mac):
 
     # Ignore testcase in Debian Jessie
     # TODO: Remove this skip if Jessie support is no longer needed
-    @pytest.mark.skipif(StrictVersion(platform.linux_distribution()[1]) <= StrictVersion('8.9'), reason="Debian 8.9 or before has no support")
+    @pytest.mark.skipif(StrictVersion(distro.linux_distribution()[1]) <= StrictVersion('8.9'), reason="Debian 8.9 or before has no support")
```

Comment:
Is this line still necessary? (probably not in scope for this PR but thought I'd ask since it's here)